app/services/idv/actions/verify_document_action.rb

```diff
@@ -0,0 +1,41 @@
+module Idv
+  module Actions
+    class VerifyDocumentAction < Idv::Steps::DocAuthBaseStep
+      def call
+        enqueue_job
+      end
+
+      private
+
+      def form_submit
+        json = form.submit
+        render_json(json, status: form.status)
+        json
+      end
+
+      def form
+        @form ||= Idv::ApiDocumentVerificationForm.new(
+          params,
+          liveness_checking_enabled: liveness_checking_enabled?,
+        )
+      end
+
+      def enqueue_job
+        document_capture_session = create_document_capture_session(
+          verify_document_capture_session_uuid_key,
+        )
+        document_capture_session.requested_at = Time.zone.now
+        document_capture_session.store_proofing_pii_from_doc({})
```

Comment:
Mentioned in slack, but reusing https://github.com/18F/identity-idp/blob/master/app/models/document_capture_session.rb#L6-L8 / https://github.com/18F/identity-idp/blob/master/app/services/document_capture_session_result.rb may be a better fit